src/Employer.Web/Controllers/NewVacancyController.cs

```diff
@@ -1,24 +1,24 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using Esfa.Recruit.Employer.Web.ViewModels.NewVacancy;
 using Esfa.Recruit.Employer.Web.Configuration.Routes;
-using Esfa.Recruit.Storage.Client.Core.Messaging;
 using Esfa.Recruit.Storage.Client.Core.Commands;
+using Esfa.Recruit.Employer.Web.Orchestrators;
 
 namespace Esfa.Recruit.Employer.Web.Controllers
 {    
     public class NewVacancyController : Controller
     {
-        private readonly IMessaging _messaging;
+        private readonly INewVacancyOrchestrator _orchestrator;
 
-        public NewVacancyController(IMessaging messaging)
+        public NewVacancyController(INewVacancyOrchestrator orchestrator)
         {
-            _messaging = messaging;
+            _orchestrator = orchestrator;
         }
         
         [HttpGet, Route("accounts/{employerAccountId}/new-vacancy", Name = RouteNames.NewVacancy_Index_Get)]
```

Comment:
Another one for discussion but I'd vote for putting the common part of the route at the class level.